river/Output.zig

```diff
@@ -70,7 +71,9 @@ pub const PendingState = struct {
         var count: u32 = 0;
         while (it.next()) |other| {
             if (other == focused_view) break;
-            count += 1;
+            if (other.pending.tags & self.tags > 0 and !other.pending.float) {
+                count += 1;
+            }
```

Comment:
At this point, this function is a lot more specialized than the name suggests. I think we should move it to `View` and call it something like `attachIndex` or similar.

Reply:
@ifreund I am not sure I agree, `attachIndex` sounds the same as `attachAfter`. 

Other ideas
* make function `attachRelative` with an enum {above, below} in `View`
* move the `focusedIndex` out of the PendingState, but still in Output and make the name more descriptive.